docs/product-manuals/cloud-console/manage-plan/available-plans.md

```diff
@@ -3,6 +3,12 @@ id: available-plans
 title: Available plans
 ---
 
-Camunda Cloud was released to the world as Beta in September 2019. In order to sign up for Camunda Cloud, please go to [the signup page](https://accounts.cloud.camunda.io/signup).
+Camunda Cloud was released to the world as Beta in September 2019. To sign up for Camunda Cloud, visit [the sign up page](https://accounts.cloud.camunda.io/signup).
 
-There are three plans available for Camunda Cloud: Free Trial, Professional, and Enterprise. For more information on Camunda Cloud plans, please visit the Camunda Cloud [product page](https://camunda.com/products/cloud/).
+There are three plans available for Camunda Cloud:
+
+- Free Trial
+- Professional
+- Enterprise
+
+For more information on Camunda Cloud plans, visit the Camunda Cloud [product page](https://camunda.com/products/cloud/).
```

Comment:
@christinaausley can you reach out to Dan Levy about this section? I'd like to get his thoughts on a few things

* Should we change the reference to the GA release date? 
* When we talk about our plans, do we really have 3? Based on the [product page](https://camunda.com/products/cloud/) it looks like you can trial the Professional or Enterprise plans.

Reply:
@akeller Dan says 1.0 launched in May 2021; 1.1 in July. He also thinks two plans are better suited here. Essentially, two plans with a free trial available. I can adjust these changes on my end.